frontend/src/modules/WellLogViewer/settings/components/TemplateTrackSettings/SortablePlotList.tsx

```diff
@@ -0,0 +1,228 @@
+import React from "react";
+
+import { WellboreLogCurveHeader_api } from "@api";
+import { arrayMove } from "@framework/utils/arrays";
+import { Dropdown, DropdownOption } from "@lib/components/Dropdown";
+import { Label } from "@lib/components/Label";
+import { SortableList, SortableListItem } from "@lib/components/SortableList";
+import { ColorSet } from "@lib/utils/ColorSet";
+import { CURVE_COLOR_PALETTE } from "@modules/WellLogViewer/utils/logViewerColors";
+import { PLOT_TYPE_OPTIONS, makeTrackPlot } from "@modules/WellLogViewer/utils/logViewerTemplate";
+import { Delete, SwapHoriz, Warning } from "@mui/icons-material";
+import { TemplatePlotTypes } from "@webviz/well-log-viewer/dist/components/WellLogTemplateTypes";
+
+import { useAtomValue } from "jotai";
+import _ from "lodash";
+
+import { allSelectedWellLogCurvesAtom } from "../../atoms/derivedAtoms";
+import { TemplatePlotConfig } from "../../atoms/persistedAtoms";
+import { AddItemButton } from "../AddItemButton";
+
+export type SortablePlotListProps = {
+    availableCurveHeaders: WellboreLogCurveHeader_api[];
+    plots: TemplatePlotConfig[];
+    onUpdatePlots: (plots: TemplatePlotConfig[]) => void;
+};
+
+export function SortablePlotList(props: SortablePlotListProps): React.ReactNode {
+    const allSelectedWellLogCurves = useAtomValue(allSelectedWellLogCurvesAtom);
+
+    const { onUpdatePlots } = props;
+
+    const curveHeaderOptions = makeCurveNameOptions(props.availableCurveHeaders);
+
+    // If the current selection does not exist, keep it in the selection, with a warning. This can happen when the user is importing a config, or swapping between wellbores
+    allSelectedWellLogCurves.forEach((curveName) => {
+        if (!curveHeaderOptions.some(({ value }) => value.endsWith("::" + curveName))) {
+            curveHeaderOptions.push(makeMissingCurveOption(curveName));
+        }
+    });
+
+    // TODO, do an offsett or something, so they dont always start on the same color?
+    const colorSet = React.useRef<ColorSet>(new ColorSet(CURVE_COLOR_PALETTE));
```

Comment:
I'll leave color selection for a later pr.... I think I'll need some bigger rewiring to make it properly synced with the workbench settings, since color, color2, fill, fill2, and colorTable all need to be set differently for different curve-types